vnext/PatchedReactNative/PatchedReactNative.vcxproj

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+ <ItemGroup Label="ProjectConfigurations">
```

Comment:
I would love it if we could share this between projects. @JunielKatarn does ARM64 build everywhere or just universal projects? I guess we could also include this everywhere, but ARM64 would fail to build if attempted on desktop.